Curatour/frontend
# Feature/dashboard event view

## *Purpose:*
*What does this merge do? Check all that apply.*


## *Description: What does this PR do?*
Adds an 'event list' component to the Dashboard.
Adds a link to the header logo that redirects to the dashboard.

## *Instructions for reviewer:*
*Where should the reviewer start?*
Lines 12-60 of `src/components/dashboard/Dashboard.js`
Lines 4 and 9 of `src/components/header/Header.js`
*How should this be manually tested?*
Pull down the branch and run `npm install`.
Open the project by running `npm start`
Navigate to `localhost:3000` and see the hard coded events displayed.
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*
![Screen Shot 2021-04-13 at 5 12 32 PM](https://user-images.githubusercontent.com/64617435/114728455-5794e700-9cfc-11eb-8c15-9552ea739ff0.png)
![Screen Shot 2021-04-14 at 8 05 34 AM](https://user-images.githubusercontent.com/64617435/114728466-5a8fd780-9cfc-11eb-829c-a9e45f1fd938.png)



## Checklist:
